lib/uds_plus_test_kit/version.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 module UDSPlusTestKit
-    VERSION = '1.1.0'
-    UDS_PLUS_VERSION = '1.1.0'
+    VERSION = '1.1.1'
+    UDS_PLUS_VERSION = '1.1.1'
```

Comment:
```suggestion
    UDS_PLUS_VERSION = '1.1.0'
```
This is for the version of the IG, which was not changed. It is ok for these 2 to be different.